### DISCUSSION TITLE
Chinese languages

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#1837] [Translations]
Right now we have many dialects for Chinese language:

- Mandarin Chinese (cmn)
- Huizhou Chinese (czh)
- Min Nan Chinese (nan)
- Chinese (zh)
- Chinese (China) (zh_CN)
- Chinese (Hong Kong) (zh_HK)
- Chinese (Taiwan) (zh_TW)
- Chinese (Singapore) (zh_SG)
- Simplified Chinese (zh-Hans)

I guess localized names of these dialects have errors, so, @sunshuwei please could you check the names and fix mistakes.

See: https://github.com/Stellarium/stellarium/blob/master/data/iso639-1.utf8

P.S. I want to get a harmonized list of the Chinese languages

P.P.S. Please welcome to discussion @coctree52 @silas1037 @dutry @superacrux @kalinysta @train11215 @freemanli - I want to hear your opinion

**Comment by silas1037** [comment]
Thanks, my idea:
cmn	Mandarin Chinese	普通话
nan	Min Nan Chinese  	闽南话
zh	Chinese	汉语
zh_CN	Chinese (China)	中文 (中国)
zh_HK	Chinese (Hong Kong)	香港繁體中文
zh_TW	Chinese (Taiwan)	臺灣正體中文
zh_SG	Chinese (Singapore)	新加坡华语
zh-Hans	Simplified Chinese	简体中文

But as I see, zh zh_CN zh-Hans cmn should be the same one.
Welcome to discuss.

**Comment by sunshuwei** [comment]
Mandarin Chinese (cmn) 官话，普通话 Mandarin only means spoken language, not script. It can be written in Simplified Chinese or Traditional Chinese. 

    Huizhou Chinese (czh) 徽语 
    Min Nan Chinese (nan) 闽南语  
    These are two Chinese dialects. They are also written in Simplified Chinese or Traditional Chinese.

    Chinese (zh) 中文 Chinese，both simplified Chinese and traditional Chinese are included. 

    Simplified Chinese (zh-Hans) 简体中文 Simplified Chinese，used in Mainland China, Malaysia and Singapore. There are a few vocabulary differences between them. 
    Chinese (China) (zh_CN) 简体中文 (中国)  Simplified Chinese，used in Mainland China
    Chinese (Singapore) (zh_SG) 简体中文 (新加坡) Simplified Chinese，used in Singapore

    Traditional Chinese (zh-Hant)  正體中文/繁體中文  Traditional Chinese used in use in Taiwan, Hong Kong and Macau. There are a few vocabulary differences between them. 
    Chinese (Hong Kong) (zh_HK) 繁體中文 (香港) Hong Kong Traditional Chinese，used in Hong Kong (and Macau)
    Chinese (Taiwan) (zh_TW) 正體中文 (臺灣) Taiwan Traditional Chinese，used in Taiwan

It is not necessary to keep so many Chinese languages. 
Just keep
zh_CN, zh_SG, zh_HK and zh_TW.
